ansible_3par_docker_plugin/tasks/deploy_FlexVolume_driver.yml

```diff
@@ -1,13 +1,16 @@
 ---
-  - name: load etcd settings
-    include_vars: '../properties/etcd_properties.yml'
-
-  - name: Install FlexVolume drivers
-    copy:
-      src: "{{ item }}"
+  - name: Download dory installer
+    get_url:
+      url: https://github.com/hpe-storage/python-hpedockerplugin/raw/master/dory_installer_v31
```

Comment:
@farhan7500  Please maintain the name dory_installer instead of dory_installer_v31. This will make sure when a new version of dory is available, the installer doesn't need to be changed.